main.rb

```diff
@@ -0,0 +1,104 @@
+require './app'
+
+def display_app
+  puts "\nPlease choose an option by enter a number:"
+  puts ['1 - List all books', '2 - List all people', '3 - Create a person',
+        '4 - Create a book', '5 - Create a rental', '6 - List all rentals for a given person id',
+        '7 - Exit']
+end
+
+def create_student(app)
+  print "\nAge: "
+  age = gets.chomp
+  print "\nName: "
+  name = gets.chomp
+  print "\nHas parent permission? [Y/N]: "
+  permission = gets.chomp
+  permission = true if 'Yy'.include?(permission)
+  permission = false if 'Nn'.include?(permission)
+  app.add_student(age.to_i, name, permission)
+end
```

Comment:
- [ ] When I try to create a student, I get this error:

![image](https://user-images.githubusercontent.com/77106962/169416421-e945da9b-d3e2-4069-b20f-2b5d71fc3696.png)

#### It has to do with the `include?` method you used. I know there are several methods to do this, one of such would be to convert the user's input to `upcase` or `downcase` before determining what `parent_permission` would be. 👍

Reply:
Thanks for informing me. 😄